docs/source/guides/getting_started/automating.rst

```diff
@@ -0,0 +1,135 @@
+Automating
+==========
+
+Automate downloading your subscriptions by running the :ref:`'sub' sub-command
+<usage:subscriptions options>` periodically. There are various tools that can run
+commands on a schedule you may use any of them that work with your installation
+method. Most users use `cron`_ in `Docker containers <docker and unraid_>`_.
+
+
+Docker and Unraid
+-----------------
+
+:doc:`The 'ytdl-sub' Docker container images <../install/docker>` provide optional cron
+support. Enable cron support by setting `a cron schedule`_ in the ``CRON_SCHEDULE``
+environment variable:
+
+.. code-block:: yaml
+  :caption: ./compose.yaml
+  :emphasize-lines: 4
+
+  services:
+    ytdl-sub:
+      environment:
+        CRON_SCHEDULE: "0 */6 * * *"
+
+Then recreate the container to apply the change and start it to generate the default
+``/config/ytdl-sub-configs/cron`` script. Read the comments in that script and edit as
+appropriate.
+
+The container cron wrapper script will write output from the cron job to
+``/config/ytdl-sub-configs/.cron.log``. The default image ``ENTRYPOINT`` will ``$ tail
+...`` that file so you can monitor the cron job in the container's output and thus also
+in the Docker logs.
+
+You may also set the ``CRON_RUN_ON_START`` environment variable to ``true`` to have the
+image run your cron script whenever the container starts in addition to the cron
+schedule.
+
+.. warning::
+
+   Using ``CRON_RUN_ON_START`` is not recommended because it may cause your cron script
+   to run too often and may trigger throttles and bans. When enabled, your cron script
+   will run *whenever* the container starts including when the host reboots, when ``#
+   dockerd`` is upgraded, when a new image is pulled, when something applies Compose
```

Comment:
dockerd?

Reply:
When the Docker daemon is restarted, such as by `# apt upgrade`.